src/main/java/com/three_tech_solutions/slot_app/dto/CreateStudentRequest.java

```diff
@@ -38,13 +34,10 @@ public class StudentRequestDTO {
     @NotNull
     private LocalDate birthday;
 
-    private LocalDate admissionDate;
+    private LocalDate admissionDate = LocalDate.now();;
 
-    //patologías no puede estar vacío
     @NotBlank
     private String pathologies;
```

Comment:
Acá estarías validando que el usuario deba ingresar que el estudiante tiene patologías, pero puede pasar que un estudiante no tenga nada, por lo que la validación está mal. Existen tres tipos de validaciones posibles 

[Docu](https://www.baeldung.com/java-bean-validation-not-null-empty-blank)
> `@NotNull`: a constrained CharSequence, Collection, Map, or Array is valid as long as it’s not null, but it can be empty.
> `@NotEmpty`: a constrained CharSequence, Collection, Map, or Array is valid as long as it’s not null, and its size/length is greater than zero.
> `@NotBlank`: a constrained String is valid as long as it’s not null, and the trimmed length is greater than zero.

Para nuestro caso si debemos aceptar valores `Null`. Para esto lo que podes hacer es cambiar la validacion a `@Size` donde podemos definir el tamaño mínimo y máximo
```suggestion
    @Size(min = 1)
    private String pathologies;
```